apis/components/v1alpha1/workbenches_types.go

```diff
@@ -36,6 +36,12 @@ type WorkbenchesCommonSpec struct {
 	// workbenches spec exposed to DSC api
 	common.DevFlagsSpec `json:",inline"`
 	// workbenches spec exposed only to internal api
+
+	// Namespace for workbenches to be installed, configurable only once when workbenches are enabled, defaults to "rhods-notebooks"
+	// +kubebuilder:default="rhods-notebooks"
+	// +kubebuilder:validation:Pattern="^([a-z0-9]([-a-z0-9]*[a-z0-9])?)?$"
+	// +kubebuilder:validation:MaxLength=63
+	WorkbenchNamespace string `json:"workbenchNamespace,omitempty"`
```

Comment:
If it's not supposed to be changed, should it have some validation like https://github.com/opendatahub-io/opendatahub-operator/blob/main/apis/dscinitialization/v1/dscinitialization_types.go#L36 ?